docs/src/inverse_problems/structural_identifiability.md

```diff
@@ -0,0 +1,168 @@
+```example si1
+si_ode = make_si_ode(goodwind_oscillator; measured_quantities=[:M])
+nothing # hide
+```
+and then used as input to various StructuralIdentifiability functions. In the following example we uses StructuralIdentifiability's `print_for_DAISY` function, printing the model as an expression that can be used by the [DAISY](https://daisy.dei.unipd.it/) software for identifiability analysis[^3].
+```example si1
+print_for_DAISY(si_ode)
+nothing # hide
+```
+
+## Notes on systems with conservation laws
+Several reaction network models, such as
+```example si2
+using Catalyst, Logging, StructuralIdentifiability # hide
+rs = @reaction_network begin
+  (k1,k2), X1 <--> X2
+end
+```
+contain conservation laws (in this case $Γ = X1 + X2$, where $Γ = X1(0) + X2(0)$ is a constant). Because the presence of such conservation laws makes structural identifiability analysis prohibitively computationally expensive (for all but the simplest of cases), these are automatically eliminated by Catalyst (removing one ODE from the resulting ODE system for each conservation law). For the `assess_identifiability` and `assess_local_identifiability` functions, this will be unnoticed by the user. However, for the `find_identifiable_functions` and `make_si_ode` functions, this may result in one, or several, parameters on the form `Γ[i]` (where `i` is an integer) appearing in the produced expressions. These correspond to the conservation law constants and can be found through
+```example si2
+conservedequations(rs)
+```
+E.g. if you run:
+```example si2
+find_identifiable_functions(rs; measured_quantities = [:X1, :X2])
+```
+we see that `Γ[1]` (`= X1(0) + X2(0)`) is detected as an identifiable expression. If we want to disable this feature for any function, we can use the `remove_conserved = false` option:
+```example si2
+find_identifiable_functions(rs; measured_quantities = [:X1, :X2], remove_conserved = false)
+```
+
+## Systems with exponent parameters
+Structural identifiability cannot currently be applied to systems with parameters (or species) in exponents. E.g. this
+```julia
+rn = @reaction_network begin
+    (hill(X,v,K,n),d), 0 <--> X
+end
+assess_identifiability(rn; measured_quantities=[:X])
+```
+is currently not possible. Hopefully this will be a supported feature in the future. For now, these expression will have to be rewritten to not include such exponents. For some cases, e.g. `10^k` this is trivial. However, it is also possible generally (but more involved and often includes introducing additional variables). If you have a system where this is required, it is recommended to contact an expert.
```

Comment:
```suggestion
is currently not possible. Hopefully this will be a supported feature in the future. For now, such expressions will have to be rewritten to not include such exponents. For some cases, e.g. `10^k` this is trivial. However, it is also possible generally (but more involved and often includes introducing additional variables). 
```
I'd just cut that last sentence. I don't think it adds anything helpful as it basically says find a generic other person to help you...